{"name":"Story 991, Test 1205 ; ValidateCallTakeCantCancelCaseWithoutCancellationReason","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1688921354382,"stop":1688921404606,"uuid":"6d68269c-d90c-46a7-82c2-e8005ee476e8","historyId":"c277972243833ab5285fe82598a6501","fullName":"Execute.CallTakerExecute.ValidateCallTakeCantCancelCaseWithoutCancellationReason","labels":[{"name":"package","value":"Execute.CallTakerExecute"},{"name":"testClass","value":"Execute.CallTakerExecute"},{"name":"testMethod","value":"ValidateCallTakeCantCancelCaseWithoutCancellationReason"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"BCAA"},{"name":"subSuite","value":"Execute.CallTakerExecute"},{"name":"host","value":"LAP10-GUYL"},{"name":"thread","value":"45940@LAP10-GUYL.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}